tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+    });
+
+    scope.$watch(tableCtrl.enableMultiSelect, function (multiple) {
+      if(tableCtrl.$$rowSelect && !multiple) {
+        // remove all but the first selected item
+        tableCtrl.selected.splice(1);
+      }
+    });
+
+    tableCtrl.registerModelChangeListener(onSelectChange);
+
+    element.on('$destroy', function () {
+      tableCtrl.removeModelChangeListener(onSelectChange);
+    });
+  }
+
+  return {
+    bindToController: true,
+    controller: Controller,
+    controllerAs: '$mdSelect',
+    link: postLink,
+    require: ['mdSelect', '^^mdTable'],
+    restrict: 'A',
+    scope: {
+      model: '=mdSelect',
+      disabled: '=ngDisabled',
+      onSelect: '=?mdOnSelect',
+      onDeselect: '=?mdOnDeselect',
+      autoSelect: '=mdAutoSelect'
+    }
+  };
+}
+
+mdSelect.$inject = ['$compile', '$parse'];
+
+angular.module('md.data.table').directive('mdTable', mdTable);
+
+function Hash() {
+  var keys = {};
+  
+  this.equals = function (key, item) {
+    return keys[key] === item;
+  };
+
+  this.get = function (key) {
+    return keys[key];
+  };
+  
+  this.has = function (key) {
+    return keys.hasOwnProperty(key);
+  };
+
+  this.purge = function (key) {
+    delete keys[key];
+  };
+  
+  this.update = function (key, item) {
+    keys[key] = item;
+  };
+}
+
+function mdTable() {
+  
+  function compile(tElement, tAttrs) {
+    tElement.addClass('md-table');
+    
+    if(tAttrs.hasOwnProperty('mdProgress')) {
+      var body = tElement.find('tbody')[0];
+      var progress = angular.element('<thead class="md-table-progress" md-table-progress>');
+      
+      if(body) {
+        tElement[0].insertBefore(progress[0], body);
+      }
+    }
+  }
+  
+  function Controller($attrs, $element, $q, $scope) {
+    var self = this;
+    var queue = [];
+    var watchListener;
+    var modelChangeListeners = [];
+    
+    self.$$hash = new Hash();
+    self.$$columns = {};
+    
+    function enableRowSelection() {
+      self.$$rowSelect = true;
+      
+      watchListener = $scope.$watchCollection('$mdTable.selected', function (selected) {
+        modelChangeListeners.forEach(function (listener) {
+          listener(selected);
+        });
+      });
+      
+      $element.addClass('md-row-select');
+    }
+    
+    function disableRowSelection() {
+      self.$$rowSelect = false;
+      
+      if(angular.isFunction(watchListener)) {
+        watchListener();
+      }
+      
+      $element.removeClass('md-row-select');
+    }
+    
+    function resolvePromises() {
+      if(!queue.length) {
+        return $scope.$applyAsync();
+      }
+      
+      queue[0]['finally'](function () {
+        queue.shift();
+        resolvePromises();
+      });
+    }
+    
+    function rowSelect() {
+      return $attrs.mdRowSelect === '' || self.rowSelect;
+    }
+    
+    function validateModel() {
+      if(!self.selected) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 16 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                if(!self.selected) {
```